lib/CL/devices/almaif/XrtDRAMRegion.cc

```diff
@@ -0,0 +1,123 @@
+/* XrtDRAMRegion.cc - accessing accelerator memory as memory mapped region.
```

Comment:
Why call it 'Xrt' perhaps you could be more thorough here in the explanation? Somehow connected to Xilinx Xrt?

Reply:
It is Xilinx xrt-specific. The comment was very outdated. Added a new comment there

Reply:
Also renamed this class to XrtExternalRegion, since this is used for DDR and HBM memories of XIlinx PCIe FPGAs

Reply:
Perhaps even add 'Xilinx' to the front of the name to make it even more explicit that this is now vendor-specific code.